inc/class-taxonomy.php

```diff
@@ -649,4 +552,31 @@ protected function upgradeChapterTypes() {
 			wp_delete_term( $type_5->term_id, 'chapter-type' );
 		}
 	}
+
+	/**
+	 * Upgrade to new Contributor Data Model
+	 *
+	 * @param int $meta_id ID of updated metadata entry.
+	 * @param int $post_id Post ID.
+	 * @param string $old_meta_key Meta key.
+	 * @param mixed $meta_value Meta value.
+	 *
+	 * @return array|false An array containing the `term_id` and `term_taxonomy_id`, false otherwise.
+	 */
+	public function upgradeToContributorTaxonomy( $meta_id, $post_id, $old_meta_key, $meta_value ) {
+		$contributor_migration = [
+			'pb_author' => 'pb_authors',
+			'pb_section_author' => 'pb_authors',
+			'pb_editor' => 'pb_editors',
```

Comment:
`pb_editor`, `pb_translator`, and `pb_contributing_authors` all save as arrays. So the `meta_value` will be an array… I believe.

Reply:
Code now handles both strings and array of strings.